nativelib/src/main/scala/scala/scalanative/meta/LinktimeInfo.scala

```diff
@@ -45,6 +45,10 @@ object LinktimeInfo {
   )
   def isWeakReferenceSupported: Boolean = resolved
 
+  @resolvedAtLinktime()
+  def isContinuationsSupported: Boolean =
```

Comment:
Not sure if this should be `areContinuationsSupported`... but then shouldn't the above check also be `areWeakReferencesSupported`?

Also, hopefully the list of platforms here is correct. I assumed so because previously only Windows was excluded.

Reply:
The plural variant seems more correct, but we won't replace the already existing `isWeakReferenceSupported` for compatibility reasons. So going with either singular (for keeping convention) or plural (for correctness) is fine with me